doc/adrs/183_add_support_for_project_dependencies.adoc

```diff
@@ -0,0 +1,40 @@
+= [ADR-183] Add support for project dependencies
+
+
+== Context
+
+https://github.com/eclipse-sirius/sirius-web/blob/master/doc/iterations/2025.2/add_support_for_the_import_of_libraries.adoc[This shape] describes how project can import libraries, but does not define how the project-to-library dependency is stored.
+
+We need to capture this information to implement future library import/upgrade strategies.
+For example: we need to know whether a part of a project comes from a library (to prevent users from editing it), or what are the libraries imported by a project that can be upgraded to a newer version.
+
+== Decision
+
+A new table `project_dependency` with the following columns should be created
```

Comment:
Since the coupling between project and semantic data has evolved in the past few weeks, this link will be necessary between semantic data and library